dlt/destinations/redshift/redshift.py

```diff
@@ -192,4 +194,3 @@ def _from_db_type(cls, pq_t: str, precision: Optional[int], scale: Optional[int]
             if (precision, scale) == cls.capabilities.wei_precision:
                 return "wei"
         return PGT_TO_SCT.get(pq_t, "text")
```

Comment:
one of the tests (direct INSERT) fails with
```
>           assert c["data_type"] == expected_c["data_type"]
E           AssertionError: assert 'time' == 'text'
E             - text
E             + time
```
this means that the redshift type for time is not `time` but most probably "TIME WITHOUT TIME ZONE" (small caps I assume)

error when copy parquet/jsonl:
```
  -----------------------------------------------
  error:  Assert
  code:      1000
  context:   Unreachable - Type not supported.
  query:     61832098
  location:  dory_type_helper.hpp:427
  process:   padbmaster [pid=1073996391]
  -----------------------------------------------
```
maybe try time with time zone? if not you can block time handling in redshift copy job (bad luck for the users) but I really hope that you can load time from files :)